### ISSUE TITLE
Process: Elaborate on MMM and Peopleware

### ISSUE THREAD

**Issue by amyjko** [amyjko/cooperative-software-development#103]
> I think the two important books: The Mythical Man Month and Peopleware: Productive Projects and Teams were not elaborated upon enough.  The paragraph ends saying they are some of the most widely read works, but why are they important?  What makes these two books compared to others so much more important and special?  I was also confused whether or not it was relevant that they were published around the same time that the Spiral model was coming into play.

**Comment by russtuck** [comment]
Also, a factual correction: The Mythical Man Month was first published in 1975.  The 1995 date is the 20th anniversary edition, with reflection on what changed and what didn't.  https://en.wikipedia.org/wiki/The_Mythical_Man-Month

**Comment by amyjko** [comment]
Correct, but with a nuance, the norm in citing academic works, is to cite the year an edition was published and then include the original publication date in a parenthetical. The publishing platform I'm using doesn't have anywhere to place this metadata, so I'll have to fix that before I can make the dates on the book clearer.

**Comment by russtuck** [comment]
It's really the text, saying the two books were published at the same time
as the Spiral model by Barry Boehm (1988).  While that's true for
Peopleware (1987), MMM was much earlier (1975).  From here, it might not
seem like 12-13 years is a lot, but given the young age of the field, these
were really different ages and stages.  MMM is mostly waterfall.  Boehm's
Spiral was a big improvement, as you pointed out.  And the MMM 20th
anniversary edition (1995) adds material and perspective that recognizes
Spiral's benefits and begins to faintly anticipate Agile.

It's a small point, which probably seems larger to me because I was in grad
school in the late 1980s, with Brooks as my dissertation advisor.  I'm
sorry if it seems pedantic.  I like your book!
